Core/VariantManager.swift

```diff
@@ -37,13 +37,9 @@ public struct Variant {
     
     public static let defaultVariants: [Variant] = [
         // SERP testing
-        Variant(name: "sc", weight: doNotAllocate, features: []),
+        Variant(name: "sc", weight: 1, features: [.onboardingCTA]),
```

Comment:
Why are we adding the onboarding CTA as a feature here? If it's a feature we want to release to all users it should just be released not made an on/off feature